### PR TITLE
Tweaks to the random trashspawner

### PR DESCRIPTION
**Pull request by SweptWasTaken**
Tones down the spawn rate a bit for both trash types. Also removes corn on the cob and empty pizza boxes from the spawn pool.

### CODE REVIEW

File: Resources/Prototypes/Entities/Effects/Markers/trash_spawner.yml [hunk 1]
```diff
@@ -17,17 +17,16 @@
     - type: TrashSpawner
       rarePrototypes:
         - PuddleVomit
-        - TrashPizzaBoxMessy
+        - PuddlePiss
+        - PuddleShit
```
Comment (ike709) [inline Resources/Prototypes/Entities/Effects/Markers/trash_spawner.yml:21]: undocumented change

do we really want this?

Comment (moonheart08) [reply Resources/Prototypes/Entities/Effects/Markers/trash_spawner.yml:21]: (The correct answer is no.)

Comment (PJB3005) [reply Resources/Prototypes/Entities/Effects/Markers/trash_spawner.yml:21]: the correct answer is, indeed, *no*

**Comment by unusualcrow** [comment]
goofball v2

**Comment by SweptWasTaken** [comment]
I removed the piss and shit. You can merge this now.

**Pull request merged**